modules/client/src/test/java/jp/co/soramitsu/iroha2/JavaTest.java

```diff
@@ -1,48 +1,36 @@
 package jp.co.soramitsu.iroha2;
 
-import jp.co.soramitsu.iroha2.client.Iroha2AsyncClient;
-import jp.co.soramitsu.iroha2.generated.Value;
-import jp.co.soramitsu.iroha2.generated.Account;
-import jp.co.soramitsu.iroha2.generated.AccountId;
-import jp.co.soramitsu.iroha2.generated.AssetId;
-import jp.co.soramitsu.iroha2.generated.AssetValue;
-import jp.co.soramitsu.iroha2.generated.AssetValueType;
-import jp.co.soramitsu.iroha2.generated.Domain;
-import jp.co.soramitsu.iroha2.generated.DomainId;
-import jp.co.soramitsu.iroha2.generated.Metadata;
-import jp.co.soramitsu.iroha2.generated.Name;
-import jp.co.soramitsu.iroha2.generated.VersionedSignedTransaction;
-import jp.co.soramitsu.iroha2.query.QueryAndExtractor;
-import jp.co.soramitsu.iroha2.query.QueryBuilder;
-import jp.co.soramitsu.iroha2.testengine.DefaultGenesis;
-import jp.co.soramitsu.iroha2.testengine.IrohaTest;
-import jp.co.soramitsu.iroha2.testengine.WithIroha;
-import jp.co.soramitsu.iroha2.transaction.TransactionBuilder;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
+import jp.co.soramitsu.iroha2.client.*;
```

Comment:
lint usually argues about wildcard imports, better to keep explicit individual ones